solutions/src/test/java/ru/sberbank/school/task02/MainTest.java

```diff
@@ -0,0 +1,61 @@
+package ru.sberbank.school.task02;
+
+import org.junit.Test;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+
+    @Test
+    public void volumeIsOverlap() {
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        ExtendedFxConversionService extendedCalculator =
+                calculatorFactory.getExtendedFxConversionService(new ExternalQuotesServiceDemo());
+        assertNotNull(extendedCalculator);
+
+        Optional<BigDecimal> priceRUB = extendedCalculator.convertReversed(ClientOperation.SELL, Symbol.USD_RUB,
```

Comment:
Abbreviation in name &#39;priceRUB&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a930033057>*